doc/api/stream.md

```diff
@@ -341,136 +553,140 @@ reader.on('end', () => {
 });
 ```
 
-Note that [`process.stderr`][] and [`process.stdout`][] are never closed until
-the process exits, regardless of the specified options.
+*Note*: The [`process.stderr`][] and [`process.stdout`][] Writable streams are
+never closed until the Node.js process exits, regardless of the specified
+options.
```

Comment:
Another note: if the reader emits an error during processing, the _`writable` will not be automatically closed_. In certain cases this can make it easy to leak memory:

```
const http = require('http')
const fs = require('fs')
http.createServer((req, resp) => {
  req.pipe(fs.createWriteStream('file')) // if "req" errors, the fs.WriteStream will never be gc'd
})
```

cc'ing @nodejs/streams to check me on this — I get this mixed up between `req.pipe(fs.WriteStream)` and `fs.createReadStream().pipe(resp)` — I'm pretty sure the former leaks, but would love a second set of :eyes:.